articles/container-instances/container-instances-tutorial-deploy-app.md

```diff
@@ -47,7 +47,14 @@ Container registry password:
 az acr credential show --name <acrName> --query "passwords[0].value"
 ```
 
-To deploy your container image from the container registry with a resource request of 1 CPU core and 1 GB of memory, run the following command. Replace `<acrLoginServer>` and `<acrPassword>` with the values you obtained from the previous two commands. Replace `<acrName>` with the name of your container registry.
+Your application will need to have
+been [prepared in advance][prepare-app]; to deploy your container image from the container registry with a
+resource request of 1 CPU core and 1 GB of memory, run the following
+command [`az container create`][az-container-create]. Replace
+`<acrLoginServer>` and `<acrPassword>` with the values you obtained
+from the previous two commands. Replace `<acrName>` with the name of
+your container registry; you can also replace `aci-tutorial-app` by
```

Comment:
"with the name of"

(instead of "by the name of")

Reply:
:+1: